### DISCUSSION TITLE
Feature Request: Support for the MGRS tile grid

### DISCUSSION THREAD

**Discussion by forrestfwilliams** [developmentseed/morecantile#149] [Q&A] [answered]
Hello, morecantile looks like a very useful package which I would ideally like to use in combination with [rio-tiler](https://cogeotiff.github.io/rio-tiler/) to split my image datasets into [Military Grid Reference System (MGRS)](https://en.wikipedia.org/wiki/Military_Grid_Reference_System) tiles. This format is the base gridding system for many remotes sensing datasets, including the Sentinel-2 optical satellite mission.

Is MGRS support something that morecantile could offer in the future, and if so what can I do to contribute?

**Comment by vincentsarago** [comment, marked as answer]
Hi @forrestfwilliams thanks for opening this discussion.

Sadly the MGRS grid do not follow the TileMatrixSet speficiation, so there is no real way to define it using Morecantile 

worth noticing that the Sentinel 2 mission does not strictly follow the MGRS grid https://hls.gsfc.nasa.gov/products-description/tiling-system/ 

note: in https://github.com/developmentseed/titiler-digitaltwin/blob/main/titiler_digitaltwin/reader.py we've used a geojson of the MGRS grid

**Reply by forrestfwilliams** [reply]
Thanks @vincentsarago this is a helpful answer. Bummer that MGRS doesn't work well with more standardized tiling schemas.
